handbook/marketing/README.md

```diff
@@ -196,7 +196,7 @@ You will need to install the following tools to use it:
 
 There are several locations in Fleet's public and internal documentation that can be helpful when answering questions raised by the community:
 
-1. Find the frequently asked question (FAQ) documents in each section in the `/docs` folder. These documents are the [Using Fleet FAQ](./../../docs/Using-Fleet/FAQ.md), [Deploying FAQ](./../../docs/Deploying/FAQ.md), and [Contributing FAQ](./../../docs/Contributing/FAQ.md).
+1. Find the frequently asked question (FAQ) documents in each section in the `/docs` folder. These documents are the [Get started FAQ](./../../docs/get started/FAQ.md), and [Contributing FAQ (on GitHub)](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/FAQ.md).
```

Comment:
This link isn't converting to HTML properly:
<img width="422" alt="Screenshot 2023-07-26 at 3 29 49 PM" src="https://github.com/fleetdm/fleet/assets/3065949/9294bd99-1a76-4374-862a-211418279f18">

Reply:
Fixed! Thank you!